content/src/debug.js

```diff
@@ -66,24 +56,64 @@ var debug = window.ide = {
     return false;
   },
   reportEvent: function(name, data) {
+
     if (!targetWindow) {
       return;
     }
-    // Based on profiling data, we dispatch by hand instead of with
-    // jQuery.trigger.  (This speeds up fractals by 40%.)
-    if (name == 'enter') { debugEnter.apply(null, data); }
-    else if (name == 'exit') { debugExit.apply(null, data); }
-    else if (name == 'appear') { debugAppear.apply(null, data); }
-    else if (name == 'resolve') { debugResolve.apply(null, data); }
-    else if (name == 'init') { debugInit.apply(null, data); }
-    else if (name == 'error') {
-      debugError.apply(null, data);
 
+    if (name == "seeeval"){
+      currentDebugId += 1;
+      record = {seeeval: true};
+      debugRecordsByDebugId[currentDebugId] = record;
+      return;
+    }
+
+    if (name == "appear"){
```

Comment:
Switch these to the old form, with a separate function for each one of the choices, for example, if (name == 'appear') { debugAppear.apply(null, data); }